gateway/engine/beans/pom.xml

```diff
@@ -12,6 +12,10 @@
 
   <dependencies>
     <dependency>
+      <groupId>net.openhft</groupId>
+      <artifactId>zero-allocation-hashing</artifactId>
+    </dependency>
+    <dependency>
```

Comment:
@msavy @EricWittmann  It appears to be Apache 2.0 licensed and a Maven build so its fine. Thanks for asking.